workflow/controller/controller.go

```diff
@@ -992,6 +1002,40 @@ func (wfc *WorkflowController) checkRecentlyCompleted(key string) bool {
 	return recent
 }
 
+func (wfc *WorkflowController) addWorkflowInformerInitHandlers() error {
```

Comment:
I'm not sure why you need another event handler. This is also permanently running, not just during init. That means that `initManagers` will be constantly called while the Controller is running

Reply:
Indeed, do you have any suggestions for modifications that can ensure throttling is not skipped while also ensuring that the data read is from the cache?